scripts/inject-cli-as-overload.py

```diff
@@ -76,37 +87,66 @@ def fill_overload(
     return_type,
     filepath,
     overload_fn,
+    class_method,
     indent=' ' * 4,
```

Comment:
wait, isn't the modification the same as `fill_overload(..., indent=' '*2)`?

Reply:
yes you're right, working on it now

Reply:
@hanxiao it's not only about `indent`, the script will also insert a `self` inside the method etc..this is what happened after i change `indent = ' ' * 2`:

![C48D9157-14A7-40FA-88AF-8367A1A16BC8](https://user-images.githubusercontent.com/9794489/121356335-09354a80-c931-11eb-8069-d90301445d9d.png)